.github/workflows/config/mlc_config.json

```diff
@@ -47,5 +47,6 @@
     ],
     "retryOn429": true,
     "fallbackRetryDelay": "30s",
+    "aliveStatusCodes": [200, 403],
```

Comment:
@cpholguera would suggest to keep it as 200 only. 403 is caused on specific links as they sit behind cloudflare or simmilar checks and they cause the test page to appear. blanket marking 403 as alive would pose different challenges for us.